umap/static/umap/js/umap.js

```diff
@@ -192,16 +192,15 @@ L.U.Map.include({
       this
     )
 
-    let isDirty = false // global status
+    let isDirty = false // self status
     try {
       Object.defineProperty(this, 'isDirty', {
         get: function () {
-          return isDirty || this.dirty_datalayers.length
+          return isDirty
         },
         set: function (status) {
-          if (!isDirty && status) self.fire('isdirty')
           isDirty = status
-          self.checkDirty()
+          this.checkDirty()
```

Comment:
Not sure this change is harmless. It works in our browser, but may it break on some older versions ?